localstack/services/apigateway/helpers.py

```diff
@@ -1241,3 +1241,54 @@ def _params(name=None):
             input_ctx["params"] = _params
             data = aws_stack.render_velocity_template(template, input_ctx, variables=variables)
     return data
+
+
```

Comment:
This code will be refactored to be applied to all integrations - it's just fixes the issue with the customer.